js/src/lib/components/InferenceWidget/shared/WidgetTableInput/WidgetTableInput.svelte

```diff
@@ -62,37 +62,39 @@
 </script>
 
 <div class="overflow-auto" bind:this={tableContainerEl}>
-	<table class="table-question-answering">
```

Comment:
Despite the appearance, all I did was put the `<table>` element between an if block:
```
{#if table.length > 1}
    [...]
{/if}
```